modules/acquia_cms_event/acquia_cms_event.info.yml

```diff
@@ -4,6 +4,7 @@ description: 'Provides an Event content type and related configuration.'
 type: module
 core_version_requirement: ^9.4 || ^10 || ^11
 dependencies:
+  - acquia_cms_common:acquia_cms_common
```

Comment:
**FYI:** The dependency on the `acquia_cms_common` module will be removed for content model to function independently in separate PR.
Same apply for all other content type modules.